modules/light-clients/06-solomachine/types/client_state.go

```diff
@@ -39,6 +39,20 @@ func (cs ClientState) GetLatestHeight() exported.Height {
 	return clienttypes.NewHeight(0, cs.Sequence)
 }
 
+// GetTimestampAtHeight returns 0.
+func (cs ClientState) GetTimestampAtHeight(
+	_ sdk.Context,
+	clientStore sdk.KVStore,
+	cdc codec.BinaryCodec,
+	height exported.Height,
+) (uint64, error) {
+	consensusState, err := GetConsensusState(clientStore, cdc, height)
+	if err != nil {
+		return 0, err
```

Comment:
Is there a reason not to return the error wrapped like you do for the tendermint client (i.e. `sdkerrors.Wrapf(err, "height (%s)", height,)`)?